packages/hadron-plugin-manager/lib/plugin.js

```diff
@@ -72,7 +73,7 @@ class Plugin {
     if (CACHE.hasOwnProperty(this.pluginPath)) {
       return CACHE[this.pluginPath];
     }
-    const module = require(path.resolve(this.pluginPath, this.metadata.main));
+    const module = req(path.resolve(this.pluginPath, this.metadata.main));
```

Comment:
This always resolves to the actual Node.js require-from-disk method, right? But the goal is still to eventually bundle all of Compass as a whole? (Or is that already happening in this PR?)

Reply:
Yep, you're right, it does the actual node resolution. Next step (COMPASS-5048) we will make all our plugins pass through the webpack build instead of that, but this logic will stay for external / third-party ones at least for now as we are not planning to completely drop support for that.

Reply:
Okay, thanks for clarifying that, totally makes sense :)